lucetc/tests/wasm.rs

```diff
@@ -2,15 +2,35 @@ use lucet_module::bindings::Bindings;
 use std::collections::HashMap;
 use std::path::PathBuf;
 
-pub fn load_wat_module(name: &str) -> Vec<u8> {
+struct LoadOptions {
+    write_debug_names: bool,
+}
+
+impl Default for LoadOptions {
+    fn default() -> Self {
+        LoadOptions {
+            write_debug_names: false,
+        }
+    }
+}
+
+fn load_wat_module(name: &str, options: LoadOptions) -> Vec<u8> {
     use std::fs::File;
     use std::io::Read;
-    use wabt::wat2wasm;
+    use wabt::Wat2Wasm;
     let watfile = PathBuf::from(&format!("tests/wasm/{}.wat", name));
     let mut contents = Vec::new();
     let mut file = File::open(&watfile).expect("open module file");
     file.read_to_end(&mut contents).expect("read module file");
-    wat2wasm(contents).expect("convert module to wasm binary format")
+    let mut wat2wasm = Wat2Wasm::new();
+    if options.write_debug_names {
+        wat2wasm.write_debug_names(true);
+    }
+    wat2wasm
+        .convert(contents)
+        .expect("convert module to wasm binary format")
+        .as_ref()
+        .to_owned()
```

Comment:
:point_up: here, we're basically writing the equivalent to the `wat2wasm` function we were importing before. You can see the definition of that function [here](https://github.com/pepyakin/wabt-rs/blob/master/src/lib.rs#L1040).